indico/modules/events/abstracts/operations.py

```diff
@@ -62,3 +63,41 @@ def delete_abstract(abstract, delete_contrib=False):
     logger.info('Abstract %s deleted by %s', abstract, session.user)
     abstract.event_new.log(EventLogRealm.management, EventLogKind.negative, 'Abstracts',
                            'Abstract "{}" has been deleted'.format(abstract.title), session.user)
+
+
+def judge_abstract(abstract, abstract_data, judgment, merge_persons=False):
+    if judgment not in AbstractAction:
```

Comment:
To prevent populating `abstract` if no judgment is passed. OTOH, now we have constraints that will prevent crap, so I guess I can get rid of this.